src/cpp/client/secure_credentials.cc

```diff
@@ -228,9 +228,10 @@ int MetadataCredentialsPluginWrapper::GetMetadata(
   }
   if (w->plugin_->IsBlocking()) {
     // Asynchronous return.
-    w->thread_pool_->Add(
-        std::bind(&MetadataCredentialsPluginWrapper::InvokePlugin, w, context,
-                  cb, user_data, nullptr, nullptr, nullptr, nullptr));
+    w->thread_pool_->Add([=]() {
```

Comment:
The empty `()` are unnecessary and can be left out. 
```suggestion
    w->thread_pool_->Add([=] {
```

Same thing below.

Reply:
I agree with this comment.

Reply:
But see my overall request about anonymous captures.

Reply:
done!